Automattic/wp-calypso
# Add Event Countdown and Timeline blocks

#### Changes proposed in this Pull Request

Adds two additional blocks to FSE plugin. These are being included here for distribution purposes and will move to Jetpack when ready the proper block categorization exists there. See https://github.com/Automattic/jetpack/pull/14104

Note: Do not deploy to WPCOM without disabling existing blocks there first, same diff commit would be ideal.

#### Testing instructions

* Build Full Site Editing Plugin
* Check you can add Event Countdown block
* Check you can add Timeline block
